frontend/taipy-gui/src/components/Taipy/DateSelector.tsx

```diff
@@ -80,6 +80,16 @@ const DateSelector = (props: DateSelectorProps) => {
         [updateVarName, dispatch, withTime, propagate, tz, props.onChange, module],
     );
 
+    const handleConsoleLog = () => {
```

Comment:
remove this ?

Reply:
I've removed it. Added for testing but I forgot to remove this file from the PR